vulnerabilities-CVEd/CVE-2022-22965-spring_data_binding_rce.bcheck

```diff
@@ -0,0 +1,21 @@
+metadata:
+    language: v1-beta
+    name: "CVE-2022-22965 Spring Data Binding RCE"
+    description: "find CVE-2022-22965 Spring Data Binding RCE"
+    author: "timeshatter"
+    tags: "Spring Data Binding RCE","CVE-2022-22965"
+
+define:
+    poc = `class.module.classLoader.URLs%5B0%5D=0`
+
+given request then
+    send request called check :
```

Comment:
Can we check the base response before issuing the request?

Also could we include an additional 'control' payload to detect if an app is simple returning a 400 when it receives an unexpected parameter? E.g. we could check the base response, issue your payload, then issuing the control which is similar but not semantically meaningful and only report if the latter does not also return a 400.

Reply:
Your idea is great, thank you. I will make the following changes